storybook/cypress/e2e/map-controls.cy.ts

```diff
@@ -52,4 +61,87 @@ describe("Map Controls", () => {
 
     cy.percySnapshot();
   });
+
+  it("should emit events", () => {
```

Comment:
Zouden dit niet eigenlijk meerdere tests moeten zijn?

Reply:
Het gaat er nu in deze change toch eigenlijk alleen maar om dat we checken op 
`.should("deep.include", { open: true })` resp `.should("deep.include", { open: false });` ?

Reply:
Ik heb de `"should emit events"` gesplitst naar 4 verschillende tests. Verder lijken mij de `should`'s die er staan wel relevant.